operators/pkg/controller/elasticsearch/mutation/comparison/config.go

```diff
@@ -7,10 +7,11 @@ package comparison
 import (
 	"fmt"
 
+	commonsettings "github.com/elastic/cloud-on-k8s/operators/pkg/controller/common/settings"
```

Comment:
alias is a bit long. No sure if I know a better alternative though 🤷‍♂ Maybe just `common`. BTW I don't like our `common` package name very much. It is as bad as `util`, `base` and friends. But that is a different question unrelated to this PR.

Reply:
I hesitate... yes, a bit long. Go for `common`.